frontend/src/containers/about/logo/index.tsx

```diff
@@ -10,9 +10,9 @@ const Logo: React.FC<LogoProps> = ({ logo: logoKey }) => {
   const { logo, alt, dimensions } = LOGOS[logoKey];
 
   return (
-    <div>
+    <span>
```

Comment:
Why not fragment?

Reply:
In most if not all cases in the About page, the parent component will be `flex`. With a fragment (or returning `Image` directly) will cause the logo to be stretched to fill the parent container.